elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/bad_join_model/models/tables/table1.hjson

```diff
@@ -0,0 +1,25 @@
+{
+  tables: [{
+      name: PlayerStats
+      sql: PlayerStats
+      schema: gamedb
+      cardinality : large
+      hidden : false
+      readAccess : A user is admin or is a player in the game
```

Comment:
For any HJSON file that we expect to fail, let's add comments explaining the line we expect to blow up and why.